v2/js/screens/loginscreen.js

```diff
@@ -166,6 +167,12 @@ const LoginScreen = {
 		}
 	},
 
+	computed: {
+		minChoosePassword() {
+			return this.l10n.stringChoosePassword.replace('{{min}}', this.min);
+		},
+	},
+
```

Comment:
Create a dedicated computed property is not the best way to handle localization here

Reply:
I will keep that in mind for the next time
Thanks for saying that